### PR TITLE
Turn on Flake8

### PR DESCRIPTION
**Pull request by chrispap95**
**Edited:** This PR brings support for flake8 linting and resolves any issues detected by the tool.
I tested the main workflow: 

1. run the processor SUEP_workflow.py
2. make plots using make_plots.py

and everything seems fine. (minus some python 3.9 features required for systematics calculation that I can't test because of my python version)

### CODE REVIEW

**Review by lucalavezzo — CHANGES_REQUESTED** [review]
looks good to me, I have a couple minor comments but nothing really important.
as an aside, I would like to move some of the parts of make_plots into another script to make the main script more readable

File: plotting/make_plots.py [hunk 1]
```diff
@@ -361,204 +573,126 @@ def create_output_file(label, abcd):
...
+                trig_weights_down,
+            )
+            df["event_weight"] *= trigSF
+
+            # 3) PS weights
+            if "PSWeight" in syst and syst in df.keys():
+                df["event_weight"] *= df[syst]
+
+            # 3) prefire weights
+            # if options.era == 2016 or options.era == 2017:
```
[the remark this answers is not in the document]
Comment (chrispap95) [reply plotting/make_plots.py:637]: The prefire weights section is already commented out in the code. I am not sure why. But I can reenable it if you want.

File: workflows/ML_coffea.py [hunk 2]
```diff
@@ -128,7 +128,7 @@ def process(self, events):
 
         # need to add these to dataframe when no events pass to make the merging work
         # for some reason, initializing these as empty and then trying to fill them doesn't work
-        columns = ["SUEP_nconst_CL", "SUEP_S1_CL"]
+        # columns = ["SUEP_nconst_CL", "SUEP_S1_CL"]
```
[the remark this answers is not in the document]
Comment (chrispap95) [reply workflows/ML_coffea.py]: done

File: workflows/CMS_corrections/track_killing_utils.py [hunk 3]
```diff
@@ -18,7 +18,7 @@ def track_killing(self, tracks):
         block1_percent = year_percent[str(self.era)]
         block2_percent = 0.01
 
-    block0_indices = tracks.pt <= 1
+    # block0_indices = tracks.pt <= 1
```
[the remark this answers is not in the document]
Comment (chrispap95) [reply workflows/CMS_corrections/track_killing_utils.py]: done

**Comment by chrispap95** [comment]
Btw, I am also pushing some changes to the dask runner so that we kill two birds with one stone (otherwise I would open a new PR very soon).

**Comment by lucalavezzo** [comment]
Changed the target branch to be working branch, instead of master.
Master should be freezed until paper is done.

File: plotting/make_plots.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (chrispap95) [reply plotting/make_plots.py:637]: Unfortunately, I have trouble running the systematics at the moment. I can't verify. I will try to find some workaround.

[the remark this answers is not in the document]
Comment (lucalavezzo) [reply plotting/make_plots.py:637]: I would say to remove the comment; I just tried it and everything ran fine.

[the remark this answers is not in the document]
Comment (chrispap95) [reply plotting/make_plots.py]: done

**Review by lucalavezzo — APPROVED** [review]

**Pull request merged**